packages/metro-babylon7/src/index.js

```diff
@@ -0,0 +1,3 @@
+const babylon = require('babylon');
+
+module.exports = babylon;
```

Comment:
Can you please add the license header to the file?

```
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 * @format
 */
```

Reply:
Fixed!